pahfit/base.py

```diff
@@ -564,3 +569,69 @@ def read(self, filename, tformat=None):
         readout = (bb_info, df_info, h2_info, ion_info, att_info)
 
         return readout
+
+    def starting_point(self, obs_x, obs_y, param_info):
+        """
+        return updated starting point in param_info based on the input spectrum
+        """
+
+        # guess starting point of bb
+        sp = interpolate.splrep(obs_x, obs_y)
+        for i, (fix, temp) in enumerate(zip(param_info[0]['amps_fixed'], param_info[0]['temps'])):
+
+            if (fix is False) & (i == 0):  # BB0 - stellar comoponent measured at 5.5 um
```

Comment:
There is no guarantee that the stellar BB component will be at the beginning, no?  Or am I wrong?  If not, then hopefully there is a way to identify the stellar component other than i==0.

Reply:
I've defined the stellar component to be BB that has temperature >= 2500K. We can perhaps change it to other temp if needed.